app/views/conference.scala.html

```diff
@@ -77,4 +77,16 @@ <h3>Administration</h3>
             </div>
         }
     </div>
+
+    <!-- The conference info panel -->
+    @if(conference.info != null) {
```

Comment:
Have you actually checked whether this is working? I kind of doubt that it will always be null if empty. You could star be testing for such cases